.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -238,6 +238,10 @@ stages:
     - publish: $(System.DefaultWorkingDirectory)/profiler/_build/DDProf-Deploy
       displayName: Upload Windows profiler home directory
       artifact: windows-profiler-home
+      
```

Comment:
I cannot comment there but you also need to remove `- profiler/` line 73 and maybe the shared dependencies that were excluded by the tracer are used by the profiler, so you may need to change that.
I suggest that you actually take those filters out and use them somewhere in the `generate_variables` stage to compute what is a tracer and a profiler change

Reply:
> but you also need to remove - profiler/ line 73

Good point, removed at a57b0f2b3a17c1de146e308adc9090025ea38692

> I suggest that you actually take those filters out
As I've wrote in comment above, I think this is good idea, but it a big change, that should come in different PR.